che-incubator/chectl
# feat: Deprecate server:start command

Signed-off-by: Anatolii Bazko <abazko@redhat.com>



### What does this PR do?
Deprecate server:start command in favor of server:deploy


### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18057

### How to test this PR?
1. Build the project
2. Run `bin/run server:start`, check the warning
3. Run 'bin/run server:deploy` to deploy Eclipse Che

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
